src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -222,11 +224,13 @@ def _get_azure_cli_properties(self):
         set_custom_properties(result, 'ShowSurveyMessage', str(self.show_survey_message))
         set_custom_properties(result, 'RegionInput', self.region_input)
         set_custom_properties(result, 'RegionIdentified', self.region_identified)
-        set_custom_properties(result, 'EnableBrokerOnWindows', str(self.enable_broker_on_windows))
-        set_custom_properties(result, 'MsalTelemetry', self.msal_telemetry)
         set_custom_properties(result, 'SecretsWarning', _get_secrets_warning_config())
         set_custom_properties(result, 'SecretsDetected', str(self.secrets_detected))
         set_custom_properties(result, 'SecretKeys', ','.join(self.secret_keys or []))
+        # authentication-related
+        set_custom_properties(result, 'EnableBrokerOnWindows', str(self.enable_broker_on_windows))
+        set_custom_properties(result, 'MsalTelemetry', self.msal_telemetry)
+        set_custom_properties(result, 'LoginExperienceV2', str(self.login_experience_v2))
```

Comment:
Should we force `str` values? `self.enable_broker_on_windows` is also a bool and we convert it to `str` (#24304). Can we use `bool` as it is? @evelyn-ys

Reply:
`bool` may not appear correctly in the final telemetry. Using `str` is more robust.